devices.php

```diff
@@ -910,6 +910,9 @@ function buildCustomAttributes($template, $device) {
 					print "\n\t<option value=\"$dcaValue\"$selected>$dcaValue</option>";
 				}
 				echo '</select></div>';
+			} else if ($cvtype=="url") {
+				echo '<div><input type="text"',$validation,' name="',$inputname,'" id="',$inputname,'" value="',$customdata["value"],'">';
+				echo '<button type="button" onclick=window.location.href="',$customdata["value"],'" value="open">Open</button></div>';

```

Comment:
It's a good thing to add, just needs a little clean-up to be consistent
with the project.

Scott

On Tue, May 10, 2016 at 10:21 AM, Wilbur Longwisch <notifications@github.com
> wrote:

> In devices.php
> <https://github.com/samilliken/openDCIM/pull/764#discussion_r62680045>:
>
> > @@ -910,6 +910,9 @@ function buildCustomAttributes($template, $device) {
> >  					print "\n\t<option value=\"$dcaValue\"$selected>$dcaValue</option>";
> >  				}
> >  				echo '</select></div>';
> > +			} else if ($cvtype=="url") {
> > +				echo '<div><input type="text"',$validation,' name="',$inputname,'" id="',$inputname,'" value="',$customdata["value"],'">';
> > +				echo '<button type="button" onclick=window.location.href="',$customdata["value"],'" value="open">Open</button></div>';
>
> Why do this as a separate if statement instead of just an include for the
> button portion? this is duplicating the input line as well which seems like
> a waste of space. Second the value of the button hasn't been encoded to be
> translated correctly.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/samilliken/openDCIM/pull/764/files/d39a74637418baa13c7240275cca586601e48de8#r62680045>
>